src/core_plugins/kibana/public/home/components/tutorial/footer.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import {
+  Footer,
+} from './footer';
+
+test('render', () => {
+  const component = shallow(<Footer
+    url={'/app/myapp'}
+    label={'launch myapp'}
+  />);
+  expect(component).toMatchSnapshot(); // eslint-disable-line
```

Comment:
What is eslint complaining about here? Maybe you can ignore the specific rule to make it clear `// eslint-disable-line no-use-before-define`

Reply:
I am not sure why that eslint line is used. I think it is just a copy paste artifact that is all over the kibana code base. I removed it and it looks like eslint runs cleanly